tools/springboard_search.sh

```diff
@@ -106,6 +106,19 @@ function get_stack_check_off_AArch64()
 	echo "stack_chk_off=$stack_chk_off; stack_chk_len=$stack_chk_len"
 }
 
+function get_frame_pointer_X86_64()
+{
+	val=$(echo $schedule_asm | awk '{for(i = 0; i <= NF; i++) if($i == "push") {print $(i+1);break;}}')
+	if [ "$val" == "%rbp" ]; then echo "1"; else echo "0"; fi
+}
+
+function get_frame_pointer_AArch64()
+{
+	val=$(echo $schedule_asm | awk '{for(i = 0; i <= NF; i++) if($i == "stp") {print $(i+1);break;}}')
+	if [ "$val" == "x29," ]; then echo "1"; else echo "0"; fi
```

Comment:
Encode the register into hex number, or simply employ the instruction code, for example

0x55 for "push %rbp"
0x53 for "push %rbx"